Lib/argparse.py

```diff
@@ -1540,6 +1540,14 @@ def _get_positional_kwargs(self, dest, **kwargs):
             msg = _("'required' is an invalid argument for positionals")
             raise TypeError(msg)
 
+        # make sure 'store_true', 'store_false', or 'store_const' action
+        # is not specified
+        if 'action' in kwargs:
+            action = kwargs.get('action')
```

Comment:
You're now doing key search twice. Using `.get` is enough.

Reply:
Fixing that.